code/_onclick/hud/screen_objects/text_objects.dm

```diff
@@ -8,3 +8,130 @@
 	screen_loc = "CENTER-7,CENTER-7"
 	maptext_height = 480
 	maptext_width = 480
+
+#define MAX_TIMERS 4
```

Comment:
Explanation would be nice here for future coders

Reply:
This has gone through more than one re-works so there's a lot of unused stuff, and I haven't even cleaned it up yet properly as I'm still trying to make it work